### ISSUE TITLE
Fix Frontend Failing Test: paddle - loss.paddle.nn.functional.kl_div

### ISSUE THREAD

**Issue by ivy-leaves** [unifyai/ivy#27696]
### Test Details
- **Function**: `paddle.nn.functional.kl_div`
- **Library**: paddle
- **Submodule**: loss

### Test Status
- **Status**: Fail
- **Workflow**: [Link to Workflow](https://github.com/unifyai/ivy/actions/runs/7069916980/job/19246084194)
- **Test Path**: `ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py::test_paddle_kl_div`
